specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -0,0 +1,3568 @@
+          {
+            "in": "path",
+            "name": "id",
+            "description": "Id of the job",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/JobPositionDetails"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets a job position in a queue": {
+            "$ref": "./examples/Jobs_GetInQueuePosition.json"
+          }
+        }
+      }
+    },
+    "/routing/workers/{workerId}/offers/{offerId}:accept": {
+      "post": {
+        "tags": [
+          "Offers"
+        ],
+        "summary": "Accepts an offer to work on a job and returns a 409/Conflict if another agent accepted the job already.",
+        "operationId": "JobRouter_AcceptJobAction",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "offerId",
+            "description": "Id of the offer",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "workerId",
+            "description": "Id of the worker",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AcceptJobOfferResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Accept a Job Offer": {
+            "$ref": "./examples/Offers_AcceptJobOffer.json"
+          }
+        }
+      }
+    },
+    "/routing/workers/{workerId}/offers/{offerId}:decline": {
+      "post": {
+        "tags": [
+          "Offers"
+        ],
+        "summary": "Declines an offer to work on a job.",
+        "operationId": "JobRouter_DeclineJobAction",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "offerId",
+            "description": "Id of the offer",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "workerId",
+            "description": "Id of the worker",
+            "required": true,
+            "type": "string"
```

Comment:
Path parameter "workerId" should appear before "offerId" (to match order in path).